controllers/alerts/alerts.go

```diff
@@ -148,9 +150,10 @@ func createUnsafeModificationAlertRule() monitoringv1.Rule {
 			"runbook_url": unsafeModificationRunbookUrl,
 		},
 		Labels: map[string]string{
-			severityAlertLabelKey:  "info",
-			partOfAlertLabelKey:    partOfAlertLabelValue,
-			componentAlertLabelKey: componentAlertLabelValue,
+			severityAlertLabelKey:     "info",
```

Comment:
@nunnatsa Hi, In the runbook it says that this alert indicates that it may lead to unexpected results or an unstable environment and upgrade is dangerous. Why is severity only "info"?

Reply:
This alert is fired when the user is using the JSON annotation to modify one of the sub-resources CRs. We want to warn the user in this case, but on the other hand, this is a valid way to go if there is no other option. In this cases we don't want to alert with too urgent severity.